model/src/main/resources/db-update/20000-from-1.8.x-to-2.0.0.sql

```diff
@@ -188,3 +188,34 @@ BEGIN transaction;
     ALTER TABLE buildrecord ADD CONSTRAINT fk_buildrecord_norebuildcause
     FOREIGN KEY (norebuildcause_id) REFERENCES buildrecord(id);
 COMMIT;
+
+
+BEGIN transaction;
+
+    -- Verify that there are no duplicate data
+    do $$
+    declare
+        total_dupes integer;
+    begin
+        SELECT count(*)
+        INTO total_dupes
+        FROM (SELECT built_artifact_id FROM build_record_built_artifact_map GROUP BY built_artifact_id HAVING count(*) > 1) AS agg;
```

Comment:
What does "AS agg" accomplish? Is that intentional?

Reply:
Fixes `ERROR:  subquery in FROM must have an alias`

Reply:
Sure!